spotlight-api/routes/api.js

```diff
@@ -32,7 +32,7 @@ const isDemoMode = demoMode.isDemoMode();
 function responder(res) {
   return function respond(err, data) {
     if (err) {
-      res.send(err);
+      res.status(500).json({ error: err.message });
```

Comment:
Great catch. I've had an issue open in pulse dashboard that I haven't had time to look into yet, to figure out why fetch errors were not raising an error properly. You found it. 👍

Reply:
yeah we don't have much in the way of error handling in this app but if we did .... this would have been a problem, it just sends a blob of HTML with an error message in it or something like that